src/main/kotlin/gg/essential/elementa/markdown/MarkdownComponent.kt

```diff
@@ -164,8 +174,27 @@ class MarkdownComponent(
         beforeDraw(matrixStack)
 
         val drawState = DrawState(getLeft() - baseX, getTop() - baseY)
+        val parentWindow = Window.of(this)
+
+        drawables.forEach {
+
+            if (elementaDebug) {
+                drawDebugOutline(
+                    matrixStack,
+                    it.layout.left.toDouble() + drawState.xShift,
+                    it.layout.top.toDouble() + drawState.yShift,
+                    it.layout.right.toDouble() + drawState.xShift,
+                    it.layout.bottom.toDouble() + drawState.yShift,
+                    this
+                )
+            }
```

Comment:
Should probably move this to be after the `isAreaVisible` check, otherwise we're drawing outlines for stuff which we're not drawing.